patches/500_add_columns_reference_metadata_start_end_date.sql

```diff
@@ -0,0 +1,19 @@
+-- ****************************************************************************
+-- RM SQL DATABASE UPDATE SCRIPT
+-- ****************************************************************************
+-- Number: 500
+-- Purpose: Add reference, metadata, start date and end date columns
+--          to collection_exercise table
+-- Author: Leo Howard
+-- ****************************************************************************
+
+ALTER TABLE casev3.collection_exercise
+    ADD COLUMN IF NOT EXISTS reference varchar(255) NOT NULL;
+
+ALTER TABLE casev3.collection_exercise
+    ADD COLUMN IF NOT EXISTS start_date timestamp with time zone='2021-09-17T23:59:59.999Z';
```

Comment:
This won't do what we need, adding `NOT NULL` columns when there is existing data needs to be done in 3 separate steps:
1. Add the column without the `NOT NULL` constraint 
2. Data patch existing columns by calculating appropriate values or just a default value
3. Add the `NOT NULL` now no existing rows contain `null` values.